app/assets/stylesheets/govuk_publishing_components/components/_layout-for-public.scss

```diff
@@ -11,3 +11,55 @@
   margin-top: - govuk-spacing(2);
   position: relative;
 }
+
+.gem-c-layout-for-public-account-menu {
+  margin: 0 0 govuk-spacing(6) 0;
+  padding: 0;
+
+  @include govuk-media-query($from: desktop) {
+    margin-right: govuk-spacing(4);
+  }
+}
+
+// Used to determine width of blue location indicator in navigation mention
+$_current-indicator-width: 4px;
```

Comment:
non-blocker: why the `_` notation?

Reply:
Because it was copied from over [here](https://github.com/alphagov/govuk-account-manager-prototype/blob/6706030ab38d0d16a86274e39db9d61211c1849e/app/assets/stylesheets/accounts.scss#L15)  and that had the `_` notation for some reason 😄  
I can update it to be more consistent with our other sass variables.